### ISSUE TITLE
Please document how to build the project

### ISSUE THREAD

**Issue by xylo** [KnowageLabs/Knowage-Server#92]
Today I tried to install Knowage.  Since the installer crashed with an exception (bug #90), I wanted to build the project from source.  I read through the whole documentation that was referenced in the README.md, but I could not find any hint how to build the project from source.

The only hint I found was in a comment of a closed ticket:
https://github.com/KnowageLabs/Knowage-Server/issues/16#issuecomment-402528280

That's pretty unusual for an open source project.  There should be at least a short section "Build from source" which should contain something like
```
Run

cd knowage-ce-parent
mvn clean package

Follow the instructions on https://knowage-suite.readthedocs.io/en/6.4/installation-guide/manual-installation.html.
```

Moreover the documentation should also note that the DDL files are located in `knowagedatabasescripts/<dbs>/`, so that the user does not have to search for them.

**Comment by xylo** [comment]
Anyone there?  Or is this project dead?

**Comment by Redjaw** [comment]
Hello,
we are writing a new detailed guide for the building process.
It will be available with the 8.0 version release.

**Comment by xylo** [comment]
Thanks.
